ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
 \pagebreak
 \section{Introduction}
-Integrating non-emitting sources of power such as solar, wind, and nuclear provides a means of reducing greenhouse gas emissions. A Nuclear Renewable Hybrid Energy System (NRHES) is a power system which involves tightly coupling a nuclear power plant, a renewable, a battery, a natural gas plant, and an industrial process through co-locating the various facilities.  NRHES is a proposed solution for allowing a nuclear power plant to load follow with a high penetration of variable energy sources. NRHES seek to minimize emissions such that they are significantly less than a renewable which is exclusively load followed by a natural gas plant \cite{Baker2016}.  Natural gas power plants, unlike nuclear and coal power plants, were built with flexibility in mind and are the typical means of load following \cite{MITEnergyInitiative2011}.  Understanding the impact of nuclear renewable hybrid energy systems in meeting future energy demand requires developing rigorous simulations to determine the system's profitability as well as its capacity to provide reliable, clean energy. 
+Integrating non-emitting sources of power such as solar, wind, and nuclear provides a means of reducing greenhouse gas emissions. A Nuclear Renewable Hybrid Energy System (NRHES) is a power system which involves tightly coupling a nuclear power plant, a renewable, a battery, a natural gas plant, and an industrial process through co-locating the various facilities.  NRHESs are a proposed solution for allowing a nuclear power plant to fluctuate the electric output to the grid with a high penetration of variable energy sources. A NRHES seeks to minimize emissions such that they are significantly less than a renewable with load following by a natural gas plant \cite{Baker2016}.  Natural gas power plants, unlike nuclear and coal power plants, were built with flexibility in mind and are the typical means of load following \cite{MITEnergyInitiative2011}.  Understanding the impact of nuclear renewable hybrid energy systems in meeting future energy demand requires developing rigorous simulations to determine the system's economic prospects as well as its capacity to provide reliable, clean energy.
```

Comment:
Reviewer#4 Comment 13: Corrected NRHES to be a singular and have the plural form NRHESs